### DISCUSSION TITLE
doris sql查询过慢

### DISCUSSION THREAD

**Discussion by ktios** [apache/doris#36274] [A - General / Q&A]
[红框圈选处的查询条件发生改变后，导致sql查询过慢。dwd_customer_detail_rongshu表是一个聚合模型。key是id和A_card_num，没有其他任何索引。
![Uploading doris_sql.jpg…]()

**Comment by ktios** [comment]
![doris_sql](https://github.com/apache/doris/assets/13539947/1b9e45db-1108-4808-a2fe-1bc9bbfe1a45)
补充图片

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题
